### ISSUE TITLE
Focus should be on the left menu when entering the wizard

### ISSUE THREAD

**Issue by sibille** [microsoft/WindowsTemplateStudio#3926]
**Describe the bug**
Focus should be on the left menu when entering the wizard.

**To Reproduce**
1. Go to VS 2019
2. Click on "Create a new project"
3. Select any of the "Windows Template Studio" project template

-> Depending on timing focus stays on Cancel button sometimes
![image](https://user-images.githubusercontent.com/24698018/97201738-155e0480-17b3-11eb-95bc-72b6ca8baa2a.png)

**Expected behavior**
Once templates are available focus should be on left menu (1. Project type)

**Screenshots**
![image](https://user-images.githubusercontent.com/24698018/97201885-39b9e100-17b3-11eb-8eb8-b463e64156c5.png)

**Comment by sibille** [comment]
Verified in dev-nightly version:
Templates version: 0.22.20310.1
Wizard version: 0.22.20310.1
